src/modules/Radio/Radio.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import uuid from "uuid/v4";
+import {
+  StandardProps,
+  WithStylesProps,
+  ThemeStyleSheetFactory
+} from "../../types";
+import { withStyles } from "../../base";
+
+interface IProps extends StandardProps<"input"> {
+  hasError?: boolean;
+  isDisabled?: boolean;
```

Comment:
you should use "disabled" , because it is already passed as a prop for the input

Reply:
so you can remove it from IProps